src/main/scala/temple/builder/project/ProjectBuilder.scala

```diff
@@ -28,13 +29,23 @@ object ProjectBuilder {
         }
     }
 
-    val dockerfiles = templefile.services.zip(1024 until Int.MaxValue).map {
+    val servicePortIterator = templefile.services.zip(1024 until Int.MaxValue)
```

Comment:
This feels like something that should be done in the main code, and not in tests. See the suggested code snippet in the discussion tab (it was attached to outdated code)